demo/main.gd

```diff
@@ -23,3 +30,13 @@ func _on_left_hand_button_pressed(name):
 	if name == "menu_button" and scene_capture:
 		print("Triggering scene capture")
 		scene_capture.request_scene_capture()
+
+
+func _on_left_controller_fb_render_model_render_model_loaded() -> void:
+	left_hand_mesh.hide()
+	hand_tablet.reparent(left_controller_render_model)
```

Comment:
I think this should just be part of the scene too. There is no reason for the tablet to be a child of what it's a child of currently, so let's just move it in the editor permanently. Otherwise, folks may look at this demo thinking there's some reason for doing this, and copy it into their own projects :-)